### PR TITLE
POS3-123 Prototype for devnet deployment

### PR DESCRIPTION
**Pull request by Stefan-Ethernal**
Benchmark tool for the devnet
https://github.com/mrwillis/eth-txn-spam

(e)txn spam tool
Run these commands:
  - `$ ./init.sh setup` creates main spam account and validator accounts, config.json. premines main spam account
  - `$ ./init.sh docker` creates docker image and deploys smart contract to v3 instance
  - `$ ./start.sh 2 100 0` - starts two docker containers with 100 spammers each. 0 is for the first mode. There are 4 modes 
  - `$ ./stop.sh` - stops docker containers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
-Make a file `mnemonics` in this repo and populate it with 12 word mnemonics, one for each line. Fund the base accounts for each mnemonic with a bunch of ETH. You can use https://iancoleman.io/bip39/ to generate mnemonics and derive the base account. 
+[Optional]:
+    - Make a file `mnemonics` in this repo and populate it with 12 word mnemonics, one for each line. Fund the base accounts for each mnemonic  with a bunch of ETH. You can use https://iancoleman.io/bip39/ to generate mnemonics and derive the base account. 
 
-The script will top up the child accounts to 10 ETH if they are under it, so make sure the base accounts have at least `10 ETH * RANGE` (see below) on your first run.
+All the actions are done with three bash scripts `init.sh`, `start.sh` and `stop.sh`:
+  - `init.sh` - initializes and/or starts bor instances before we can do actual spamming
+      Execute `$ ./init.sh` to see all the commands and options from this init tool
+      Basic/necessary commands are:
+        `$ ./init.sh setup-bor <bor-path>` initialize account and genesis (by default) into `~/borv3/test-dir-`
```
Comment (ferranbt) [inline README.md]: Do not call it `bor` but `v3`. That is the placeholder name.

Comment (igorcrevar) [reply README.md]: fixed

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (vcastellm) [inline .vscode/launch.json]: Do we want to include this config?

Comment (igorcrevar) [reply .vscode/launch.json]: fixed

**Review by ferranbt — APPROVED** [review]

**Pull request merged**